napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -62,6 +65,23 @@ def __init__(self, viewer):
         layout.addWidget(self.newPointsButton)
         layout.addWidget(self.newShapesButton)
         layout.addWidget(self.newLabelsButton)
+        if os.environ.get('DEBUG_NAPARI_NOTIFICATION', False):
```

Comment:
do we want to keep this?

Reply:
Up to you, I think it's useful for testing even of the future Gui elements. I can move it somewhere else; or make it a plugin.

Reply:
minimally, I don't think it should be used for pytests. So, rather than setting the environment variable, etc... just _make_ a button:
```python
def raise_():
    raise ValueError("error!")

def warn_():
    import warnings

    warnings.warn("warning!")

viewer = make_napari_viewer()

errButton = QPushButton(viewer.window.qt_viewer)
warnButton = QPushButton(viewer.window.qt_viewer)
errButton.clicked.connect(raise_)
warnButton.clicked.connect(warn_)
```